app/charts/bar/bars-state.tsx

```diff
@@ -130,7 +130,8 @@ const useBarsState = (
       .paddingInner(PADDING_INNER)
       .paddingOuter(PADDING_OUTER);
     const yScaleInteraction = scaleBand()
-      .domain(bandDomain)
+      //NOTE: not sure if this is the right way to go here
+      .domain(bandDomain.reverse())
```

Comment:
Not 100% what the problem was, but maybe this could be already done one level up, for example in the `useX` / `useYBandVariables`?